### PR TITLE
updating fix-lint script in package.json

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-4849

#### Brief feature description
-

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- npn run fix-lint to test

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

**Comment by ratneshraval** [comment]
I see only 15 errors left, can you fix them :)
When done update lint command to include test folder as well.

File: package.json [hunk 1]
```diff
@@ -46,8 +46,8 @@
     "whatwg-fetch": "^2.0.3"
   },
   "scripts": {
-    "lint": "eslint --ext .jsx --ext .js app/;",
-    "fix-lint": "eslint --ext .jsx --ext .js --fix app/;exit 0;",
+    "lint": "eslint --ext .jsx --ext .js app/; eslint --ext .js --fix test/javascript/",
```
Comment (ratneshraval) [inline package.json]: you don't need --fix here in "lint" command. Also it is missing .jsx extension here and in 'fix-lint'

File: test/javascript/rfa_forms/rfa01a_edit_view/aboutThisResidenceCard_tests.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import AboutThisResidenceCard from 'rfa_forms/rfa01a_edit_view/aboutThisResidenceCard.jsx'
 import {languageTypes, residenceTypes} from './../../helpers/constants'
-var TestUtils = require('react-dom/test-utils')
 import {shallow, mount} from 'enzyme'
+var TestUtils = require('react-dom/test-utils')
```
Comment (anuroop15) [inline test/javascript/rfa_forms/rfa01a_edit_view/aboutThisResidenceCard_tests.js]: can this  import be removed as it's not being used.

Comment (Nasir2712) [reply test/javascript/rfa_forms/rfa01a_edit_view/aboutThisResidenceCard_tests.js]: done

**Pull request merged**